core/config/locales/en.yml

```diff
@@ -823,6 +832,7 @@ en:
     exchange_for: Exchange for
     expedited_exchanges_warning: "Any specified exchanges will ship to the customer immediately upon saving. The customer will be charged the full amount of the item if they do not return the original item within %{days_window} days."
     excl: excl.
+    exchange: Exchange
```

Comment:
It was a missing translation I noticed in the `order/carton` partial while I was working.  Perhaps could have just added it alone in a separate PR.